src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -84,13 +84,23 @@ public void storeMessage(Message message) {
     return messages;
   }
 
+
+
+  
       /** Returns the total number of messages for all users. */
     public int getTotalMessageCount(){
       Query query = new Query("Message");
       PreparedQuery results = datastore.prepare(query);
       return results.countEntities(FetchOptions.Builder.withLimit(1000));
     }
 
+    /** Returns the total number of users for in site. */
+    public int getTotalUserCount(){
+      Query query = new Query("User");
+      PreparedQuery results = datastore.prepare(query);
```

Comment:
nit: imo you can just move query construction to next line

Reply:
couldn't understand..